product_state/models/product_template.py

```diff
@@ -26,11 +26,13 @@ class ProductTemplate(models.Model):
         tracking=10,
     )
 
+    # Choose from state has default=True, needed override when new states implemented
     @api.model
     def _get_default_product_state_id(self):
-        return self.env.ref(
-            "product_state.product_state_sellable", raise_if_not_found=False
+        res = self.env["product.state"].search(
+            [("default", "=", True), ("active", "=", True)]
         )
+        return res[0] if res else False
```

Comment:
the "active" leaf is useless as it's done by default

```suggestion
        return self.env["product.state"].search([("default", "=", True)], limit=1).id
```